bar/bootstrap.sh

```diff
@@ -1,9 +1,20 @@
 #!/usr/bin/bash
-sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
-sudo yum-config-manager --enable docker-ce-nightly
-sudo yum-config-manager --enable docker-ce-test
-sudo yum -y install docker-ce docker-ce-cli containerd.io
+sudo yum install -q -y yum-utils
+sudo yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+yum -y install docker docker-ce docker-ce-cli containerd.io
+if [[ $? -eq 0 ]]; then
```

Comment:
yum can produce all kind of statuses which are unrelated to the fact if docker is installed or not. You might want to use a different check for that.